cmake/TaichiCore.cmake

```diff
@@ -38,7 +38,7 @@ endif()
 
 option(USE_STDCPP "Use -stdlib=libc++" OFF)
 option(TI_WITH_CUDA "Build with the CUDA backend" OFF)
-option(TI_WITH_OPENGL "Build with the OpenGL backend" OFF)
+option(TI_WITH_OPENGL "Build with the OpenGL backend" ON)
```

Comment:
I would set it to OFF to let CI pass for now. Please open up another PR to add the GLEW/GLFW submodules and make buildbots happy. After that, I'll enable OpenGL building & testing on the buildbots at MIT with GPUs.

Reply:
Not necessary, this will be OFF if GLEW not detected on current machine.

Reply:
I see. Sounds good.